spec/models/manageiq/providers/vmware/infra_manager/inventory/collector_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'rbvmomi/vim'
+require 'rbvmomi'
```

Comment:
The base `rbvmomi` module requires `rbvmomi/connection`, `rbvmomi/sso`, and `rbvmomi/vim` but not `pbm` and `sms` which was the point of only requiring `rbvmomi/vim` here.  Now `rbvmomi/vim` depends on `rbvmomi/connection` so we should really be requring the base